service-commands/src/commands/service-commands.json

```diff
@@ -140,7 +140,13 @@
     ],
     "down": [
       "cd discovery-provider; . compose/env/tmp/shellEnv#.sh; docker-compose -f compose/docker-compose.db.yml -f compose/docker-compose.redis.yml -f compose/docker-compose.backend.yml -f compose/docker-compose.ipfs.yml down",
-      "cd discovery-provider; . compose/env/unsetShellEnv.sh"
+      "cd discovery-provider; . compose/env/unsetShellEnv.sh",
```

Comment:
can we also run this on A down? i think it only runs on discovery down right now?

Reply:
Good catch and sure thing! Please check the latest changes.